framework/source/class/qx/ui/table/cellrenderer/Abstract.js

```diff
@@ -223,6 +223,8 @@ qx.Class.define("qx.ui.table.cellrenderer.Abstract",
         'left:', cellInfo.styleLeft, 'px;',
         this._getCellSizeStyle(cellInfo.styleWidth, cellInfo.styleHeight, this._insetX, this._insetY),
         this._getCellStyle(cellInfo), '" ',
+        'data-qx-row="', cellInfo.row, '" ',
+        'data-qx-col="', cellInfo.col, '" ',
```

Comment:
This will be useful. Thanks! I think, though, since these are qx-generated attributes, the attribute names should begin with `qx` and should be descriptive and unambiguous. I'd suggest `qx-table-cell-row` and `qx-table-cell-col`.

Reply:
Good idea, I support that. This will make clear that it is part of a table.

Reply:
Sorry, didn't read close enough - the attributes should really begin with "data"  to follow the specs and to support the dataset API:

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*
https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/dataset

Reply:
So the unambiguous attribute name would be `data-qx-table-cell-(row|col)`.

Reply:
I wasn't aware of that standard. Based on that, I agree with @cboulanger

Reply:
Ok, I've updated the attribute names accordingly